api/src/main/java/org/hawkular/inventory/api/model/ResourceType.java

```diff
@@ -39,19 +40,19 @@
     private ResourceType() {
     }
 
-    public ResourceType(String tenantId, String id) {
-        super(tenantId, id);
+    public ResourceType(CanonicalPath path) {
+        super(path);
```

Comment:
takze tady ty zmeny tech signatur konstruktoru a na tech dalsich entitach to taky vynucuje pak tu zmenu v `hawkular/hawkular`